Add information on multi-db options to the CodeQL CLI docs (#6153)
github/codeql: docs/codeql/codeql-cli/creating-codeql-databases.rst

- Check out the version of your codebase you want to analyze. The directory
  should be ready to build, with all dependencies already installed.

> For information about using the CodeQL CLI in a CI system to create results

Reviewer: Should we specifically mention "aside from GitHub Actions"?

Author: Good point - we probably should be explicit here.

Author: ```suggestion
For information about using the CodeQL CLI in an third-party CI system to create results
```

to display in GitHub as code scanning alerts, see `Configuring CodeQL CLI in your CI system <https://docs.github.com/en/code-security/secure-coding/using-codeql-code-scanning-with-your-existing-ci-system/configuring-codeql-cli-in-your-ci-system>`__ 

> in the GitHub documentation.

Author: ```suggestion
in the GitHub documentation. For information about enabling CodeQL code scanning using GitHub Actions,
see `Setting up code scanning for a repository <https://docs.github.com/en/code-security/secure-coding/automatically-scanning-your-code-for-vulnerabilities-and-errors/setting-up-code-scanning-for-a-repository>`__ 
in the GitHub documentation.
```

  be created when you execute the command---you cannot specify an existing
  directory. 
- ``--language``: the identifier for the language to create a database for.
  When used with ``--db-cluster``, the option accepts a comma-separated list, 
  or can be specified more than once.
  CodeQL supports creating databases for the following languages:

  .. include:: ../reusables/extractors.rst

You can specify additional options depending on the location of your source file, 
if the code needs to be compiled, and if you want to create CodeQL databases for 
more than one language:

- ``--source-root``: the root folder for the primary source files used in
  database creation. By default, the command assumes that the current
  directory is the source root---use this option to specify a different location.
- ``--db-cluster``: use for multi-language codebases when you want to create
  databases for more than one language. 
- ``--command``: used when you create a database for one or more compiled languages,

>   omit if the only langauges requested are Python and JavaScript. 

Reviewer: ```suggestion
  omit if the only languages requested are Python and JavaScript. 
```

build system.

When the database is successfully created, you'll find a new directory at the
path specified in the command. If you used the ``--db-cluster`` option to create

> more than one database, a subdirectory is created for each langauge.

Reviewer: ```suggestion
more than one database, a subdirectory is created for each language.
```

.. pull-quote:: Important

   When you run ``database create`` for JavaScript, TypeScript, and Python, you should not
   specify a ``--command`` option. Otherwise this overrides the normal
   extractor invocation, which will create an empty database. If you create
   databases for multiple languages and one of them is a compiled language,

>    use the ``--no-run-unnecessary-builds`` option to skip the command for these
>    languages.

Reviewer: ```suggestion
   use the ``--no-run-unnecessary-builds`` option to skip the command for the languages that don't need to be compiled.
```
